docs/start/integration.md

```diff
@@ -5,111 +5,117 @@ title: Integration Guide
+Each ATX is linked to one smesher, is identified by that smesher's `smesherID` (a 32-byte public key), and targets one epoch. If a smesher generates multiple ATXs targeting the same epoch, that smesher becomes guilty of [equivocation](#malfeasance) and its identity will be disqualified.
 
-ATXs have an associated [[weight|Smesher-Guide#atxs-and-weight]] which depends on the contents of the miner's proof of spacetime.
+ATXs have an associated [weight](../learn/atx.md/#weight) which depends on the contents of the smesher's PoST.
 
 ### Proposals and Rewards
 
-Miners that successfully establish eligibility to mine in a given epoch will be given one or more eligibility slots (i.e., layers); every miner gets at least one such slot per epoch. If the miner is online and in sync when their slot arrives they'll generate and submit a proposal to the network. The proposal contains a subset of the transactions in the miner's view of the mempool, chosen so that the union of all proposals from all miners is enough to efficiently fill a block with unique, valid transactions. (In practice, until transaction throughput increases and blocks are regularly full, proposals will include the entire contents of a miner's view of the mempool.) The [[Hare|Smesher-Guide#hare]] consensus mechanism subsequently allows the network to agree on the canonical set of proposals for a given layer, which get assembled deterministically into a block for the layer.
+Smeshers who successfully establish eligibility to smesh in a given epoch will be allotted one or more **eligibility slots** (which are layers). Every smesher gets at least one such slot per epoch. If a smesher is online and in sync when their slot arrives, they will generate and submit a **[block proposal]** to the network. The proposal contains a subset of the transactions present in the smesher's view of the **mempool** (the place where unconfirmed transactions live before being selected to be included in block proposals).
 
-This mechanism allows Spacemesh to scale transaction throughput efficiently by achieving high throughput (i.e., big blocks per layer) with small messages (i.e., proposals that can be gossiped with low latency). Full blocks are never gossiped over the network in the course of protocol execution; rather, each node deterministically computes the block for each layer on the basis of these proposals.
+These transactions are chosen in such a way that the union of all proposals from all smeshers is enough to efficiently fill a **block** (a set of confirmed transactions) with unique, valid transactions. In practice, until transaction throughput increases and blocks are regularly full, proposals will include the *entire* contents of a smesher's view of the mempool. The Hare consensus mechanism subsequently allows the network to agree on the canonical set of block proposals for a given layer, which then get deterministically assembled into a block for the layer.
```

Comment:
> 

Hmm, I think the original is still better. I want to bold "block" to signify it as an important concept.

Reply:
Sure but my change was mostly to put efficiently after "to fill a block" and not before the verb since efficiently is an adverb and should come after the verb.

Reply:
What do you think about this:

"These transactions are chosen in such a way that the union of all proposals from all smeshers is enough to fill a **block** (a set of confirmed transactions) efficiently with unique, valid transactions."

Reply:
Yes looks good